_examples/structs/test.py

```diff
@@ -53,3 +53,18 @@
 except Exception as err:
     print("caught error: %s" % (err,))
     pass
+
+class S2_descendant(structs.S2):
+    def __init__(self, n):
+        super(S2_descendant, self).__init__(n)
+        self.local = n
+
+try:
+    s2_descendant = S2_descendant(43)
+    print("s2_descendant = %s" % (s2_descendant,))
```

Comment:
what about implementing `__str__` on `S2Kid` so the printout makes more sense (and actually test "polymorphism") ?

Reply:
Good idea.